components/fxa-client/android/src/main/java/mozilla/appservices/fxaclient/Config.kt

```diff
@@ -8,59 +8,23 @@ package mozilla.appservices.fxaclient
  * Config represents the server endpoint configurations needed for the
  * authentication flow.
  */
-class Config constructor(val contentUrl: String, val clientId: String, val redirectUri: String) {
-
+class Config constructor(
+    val contentUrl: String,
+    val clientId: String,
+    val redirectUri: String,
+    val tokenServerUrlOverride: String? = null
```

Comment:
At some point this constructor might grow even more (in terms of args), but should be okay to add `tokenServerUrlOverride` for now and refactor this later if needed

Reply:
Yes if it grows more we should switch to a constructor pattern instead.